rust_dev_preview/s3/README.md

```diff
@@ -1,254 +1,76 @@
-- __-v__ displays additional information.
-
-### put-object-presigned
-
-This example uploads a file to an Amazon S3 bucket, creates a public URI to the object, and displays the URI.
-
-`cargo run --bin put-object-presigned -- -b BUCKET -o OBJECT [-e EXPIRES-IN] [-r REGION] [-v]`
-
-- _BUCKET_ is the name of the bucket.
-- _OBJECT_ is the name of the file to upload to the bucket.
-- _EXPIRES-IN_ is the duration, in seconds, that the URI is valid. The default is 900 (15 minutes).
-- _REGION_ is the Region in which the client is created.
-  If not supplied, uses the value of the __AWS_REGION__ environment variable.
-  If the environment variable is not set, defaults to __us-west-2__.
-- __-v__ displays additional information.
-
-### s3-helloworld
-
-This example lists your buckets and uploads a file to a bucket.
-
-`cargo run --bin s3-helloworld -- -b BUCKET -f FILENAME -k KEY [-r REGION] [-v]`
-
-- _BUCKET_ is the name of the bucket.
-- _FILENAME_ is the name of the file to upload.
-- _KEY_ is the name of the file to upload to the bucket.
-- _REGION_ is the Region in which the client is created.
-  If not supplied, uses the value of the __AWS_REGION__ environment variable.
-  If the environment variable is not set, defaults to __us-west-2__.
-- __-v__ displays additional information.
-
-### s3-object-lambda
+### Prerequisites
 
-This example lists your buckets in a specified endpoint.
+- You must have an AWS account, and have configured your default credentials and AWS Region as described in [https://github.com/awslabs/aws-sdk-rust](https://github.com/awslabs/aws-sdk-rust).
+- Install Rust and Cargo as described [in the Rust documentation](https://doc.rust-lang.org/book/ch01-01-installation.html).
 
-`cargo run --bin s3-object-lambda -- -a ACCOUNT -e ENDPOINT [-r REGION] [-v]`
+## Tests
 
-- _ACCOUNT_ is the your account number.
-- _ENDPOINT_ is the endpoint.
-- _KEY_ is the name of the file to upload to the bucket.
-- _REGION_ is the Region in which the client is created.
-  If not supplied, uses the value of the __AWS_REGION__ environment variable.
-  If the environment variable is not set, defaults to __us-west-2__.
-- __-v__ displays additional information.
+⚠️ Running the tests might result in charges to your AWS account.
 
-The object must have the following format.
+Run the tests with the following command:
 
 ```
-Name,PhoneNumber,City,Occupation
-Person1,(nnn) nnn-nnnn,City1,Occupation1
-...
-PersonN,(nnn) nnn-nnnn,CityN,OccupationN
+cargo test -p s3_code_examples -- --include-ignored --nocapture
 ```
 
-### select-object-content.rs
-
-This example uses an SQL query to retrive information from an object, in CSV format, in an Amazon S3 bucket.
-
-`cargo run --bin select-object-content -- -b BUCKET -o OBJECT -n NAME [-r REGION] [-v]`
-
-- _BUCKET_ is the name of the bucket.
-- _OBJECT_ is the name of the object to query.
-- _NAME_ is the name of the person to retrieve infomation about.
-
-## Resources
+## Additional resources
```

Comment:
Template says to include links to:
* [Service Developer Guide](link to developer guide)
* [Service API Reference](link to service API reference)
In addition to the SDK links.

Reply:
Added